fake/src/faker/impls/filesystem.rs

```diff
@@ -56,3 +57,26 @@ impl<L: Data> Dummy<DirPath<L>> for String {
         p.to_string_lossy().into()
     }
 }
+
+const UNSTABLE_SEMVER: &'static [&'static str] = &[
+    "alpha", "beta", "rc"
+];
+
+impl<L: Data> Dummy<Semver<L>> for String {
+    fn dummy_with_rng<R: Rng + ?Sized>(_: &Semver<L>, rng: &mut R) -> Self {
+        let patch = &mut(0..20).fake_with_rng::<u8, _>(rng).to_string();
+        if Boolean(EN, 10).fake() {
```

Comment:
should use `Boolean(EN, 10).fake_with_rng(rng)`

Reply:
done